tests/test_util.py

```diff
@@ -3,16 +3,20 @@
 import pytest
 import sfs
 
+
 cart_sph_data = [
     ((1, 1, 1), (np.pi / 4, np.arccos(1 / np.sqrt(3)), np.sqrt(3))),
-    ((-1, 1, 1), (np.arctan2(1, -1), np.arccos(1 / np.sqrt(3)), np.sqrt(3))),
-    ((1, -1, 1), (-np.pi / 4, np.arccos(1 / np.sqrt(3)), np.sqrt(3))),
-    ((-1, -1, 1), (np.arctan2(-1, -1), np.arccos(1 / np.sqrt(3)), np.sqrt(3))),
+    ((-1, 1, 1), (3 / 4 * np.pi, np.arccos(1 / np.sqrt(3)), np.sqrt(3))),
+    ((1, -1, 1), (-np.pi / 4, np.arccos(1 / np.sqrt(3)),
+                  np.sqrt(3))),
```

Comment:
Is there a reason for these line breaks?

Reply:
This was unnecessary and I missed it, thanks!
Rebased and force pushed.